### PR TITLE
Fixes #325, profilo Ludovico in inglese

### LINKED ISSUE #325: Profilo Ludovico
### Expected behavior

Profilo Ludovico esiste in inglese e italiano

### Actual behavior

Profilo Ludovico non esiste

### Steps to reproduce the behavior

Nella pagina [chi siamo](https://teamdigitale.governo.it/it/47-content.htm) e [About us](https://teamdigitale.governo.it/en/49-content.htm) deve esserci Ludovico nella lista

### PR DESCRIPTION
**Pull request by cloudify**
Fixes issue #325 

Brief description of changes proposed in this pull request:
Aggiunge profilo in inglese di @ludopoggi 

cc @teamdigitale/code-reviewers

### CODE REVIEW

File: en/people/ludovico-poggi.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+After few months, thanks to one of these policies (18app, a platform that provides a cultural bonus to the young adults turning 18), I met Diego Piacentini, the Government Commissioner for the Digital Agenda. Since the first meeting I noticed his analytical and managerial approach to problems and his ability to move forward incrementally and without any pre-conceived ideology. I am still learning this useful method and one day I would hope the entire public administration could change its paradigm and adopt the same modus operandi.
+
+In December 2016, in the aftermath of the Renzi’s government resignation, my contract expired and I discovered that Piacentini’s Digital Team was hiring: among the vacancies, I applied for the position in the Communications and institutional relations department.
```
Comment (gvarisco) [inline en/people/ludovico-poggi.md]: I'd say "I applied for the position in the Communications and institutional relations department." -> "I applied for the position in the Communications and institutional affairs department." to be aligned with the title chosen.

File: en/people/ludovico-poggi.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+medium_user:
+linkedin_url: https://www.linkedin.com/in/ludovico-poggi-212a738/
+ref: ludovico-poggi
+parent_ref: team
+start_date: February 1st, 2017
+period_provided: until September 16, 2018
+annual_compensation: € 67.000
+approved_by_court: May 3rd, 2017
+---
+I started my career at lavoce.info, an italian think tank and economic policy watch dog founded in 2002 by Tito Boeri. During this experience I learnt how to differentiate comments and facts, and used the latter to develop a solid assessment of the process of policy making, independently from the political parties that generated it. In this laboratory of ideas, where independence is of utmost importance, I managed to meet tight deadlines and covered different areas of the communication universe: press office, editing, fundraising, social media management and organization of events. I developed a learning-by-doing approach relying extensively on a wide range of instruments: internet, Twitter,(io li toglierei... in fin dei conti siamo nel 2017) a weekly newsletter, crowdfunding events and the community management.
```
Comment (gvarisco) [inline en/people/ludovico-poggi.md]: there's an italian sentence here that needs to be removed ("(io li toglierei... in fin dei conti siamo nel 2017)). I'd say 'modern instruments'.

File: en/people/ludovico-poggi.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+Whilst working at lavoce.info, I have been a free-lance journalist: as an elections reporter in Greece and Spain, an editor in chief of a web-journal focused on gender economy and as a project manager for a proposal that evaluated the use of internet among enterprises located in Lombardy. I have also worked as a consultant for the development of a Public Administration’s websites and contributed to edit a book on economic policy proposals to the national government.
```
Comment (gvarisco) [inline en/people/ludovico-poggi.md]: typo: elections reporter -> elections' reporter

File: en/people/ludovico-poggi.md (hunk 2 above)
Comment (gvarisco) [reply en/people/ludovico-poggi.md]: err, 'modern tools'.

**Comment by ludopoggi** [comment]
Title: Communication and Institutional Affairs

I started my career at lavoce.info, an italian think tank and economic policy watch dog founded in 2002 by Tito Boeri. During this experience I learnt how to differentiate comments and facts, and used the latter to develop a solid assessment of the process of policy making, independently from the political parties that generated it. In this laboratory of ideas, where independence is of utmost importance, I managed to meet tight deadlines and covered different areas of the communication universe: press office, editing, fundraising, social media management and organization of events. I developed a learning-by-doing approach relying extensively on a wide range of instruments: internet, Twitter, a weekly newsletter, crowdfunding events and the community management.

Whilst working at lavoce.info, I have been a free-lance journalist: as an election's reporter in Greece and Spain, an editor in chief of a web-journal focused on gender economy and as a project manager for a proposal that evaluated the use of internet among enterprises located in Lombardy. I have also worked as a consultant for the development of a Public Administration’s websites and contributed to edit a book on economic policy proposals to the national government.

After few months, thanks to one of these policies (18app, a platform that provides a cultural bonus to the young adults turning 18), I met Diego Piacentini, the Government Commissioner for the Digital Agenda. Since the first meeting I noticed his analytical and managerial approach to problems and his ability to move forward incrementally and without any pre-conceived ideology. I am still learning this useful method and one day I would hope the entire public administration could change its paradigm and adopt the same modus operandi.

In December 2016, in the aftermath of the Renzi’s government resignation, my contract expired and I discovered that Piacentini’s Digital Team was hiring: among the vacancies, I applied for the position in the Communications and institutional affairs department.

Start Date: February 1, 2017
Period Provided: until September 16, 2018
Annual Compensation: € 67,000
Approved by the Court of Auditors: May 3, 2017

**Pull request merged**